### ISSUE TITLE
Unable to install pysnow because pytz version dependence

### ISSUE THREAD

**Issue by pvangara** [rbw/pysnow#193]
Unable to install pysnow because of the following dependencies that we have:

pysnow 0.7.17 depends on pytz<2020.0 and >=2019.3

Where has Pandas depends on higher version:
pandas 1.4.4 depends on pytz>=2020.1

Could this be fixed as soon as possible, we are stuck. If not fixed, we have to see alternatives.

Thanks,
Ranga.

**Comment by MallocArray** [comment]
Agreed.  The fact that this is pinned to an old version of pytz is causing us issues as well

**Comment by Mariushrek** [comment]
Aloha!

Any news? ;)

Thanks,
Mariusz

**Comment by MallocArray** [comment]
I found another Closed Issue that mentions development is done with this module and only being maintained. There is a new modern module that looks to be taking its place, but it hasn't been updated for a few years either:
https://github.com/rbw/pysnow/issues/184

https://github.com/rbw/aiosnow

I was needing this for an Ansible playbook, but it appears that the Ansible collection I was using is also depreciated and replaced with a new one that doesn't list any python module requirements, so I'll be migrating to that:
https://github.com/ansible-collections/servicenow.itsm

**Comment by dcompane** [comment]
Just opened a pull request to bump the dependencies. In the meantime, I am installing the module from my fork using

pip install git+https://github.com/dcompane/pysnow.git

Feel free to check that my repo is just one commit ahead of the master, and it is only the requirements bump. Install from there if you would like, but there is no maintenance or support promised.
